src/core/components/hv-root/index.tsx

```diff
@@ -28,6 +48,460 @@ export default class Hyperview extends PureComponent<HvScreenProps.Props> {
 
   static renderElement = Render.renderElement;
 
+  behaviorRegistry: BehaviorRegistry;
+
+  componentRegistry: ComponentRegistry;
+
+  formComponentRegistry: ComponentRegistry;
+
+  parser: Dom.Parser;
+
+  constructor(props: HvScreenProps.Props) {
+    super(props);
+
+    this.onUpdate = this.onUpdate.bind(this);
+    this.reload = this.reload.bind(this);
+
```

Comment:
I know we have this in HvScreen, but this is no longer needed in modern JS (the binding already happens when we declare an instance method using the "fat arrow" syntax
```suggestion
```

Reply:
Thanks, I was very curious about what benefit it was providing. Answers online didn't provide much insight.